docs/src/app/components/pages/components/progress.jsx

```diff
@@ -72,6 +72,12 @@ const ProgressPage = React.createClass({
             header: 'optional',
             desc: 'Override the inline-styles of the progress\'s root element.',
           },
+          {
+            name: 'color',
```

Comment:
@oliviertassinari I added examples in the most recent commit. All your notes have been address, let me know if you have any changes you'd like to see made to the examples.